lib/awful/widget/taglist.lua

```diff
@@ -177,6 +177,10 @@ function taglist.new(screen, filter, buttons, style, update_function, base_widge
     local uf = update_function or common.list_update
     local w = base_widget or fixed.horizontal()
 
+    if w.set_spacing and (style and style.spacing or beautiful.taglist_spacing) then
```

Comment:
Doesn't this now also need a doc comment? Something like `@beatiful taglist_spacing`?